editor/src/components/canvas/commands/add-imports-to-file-command.ts

```diff
@@ -31,7 +31,12 @@ export const runAddImportsToFile: CommandFunction<AddImportsToFile> = (
     command.targetFile,
     editorState,
     (parseSuccess) => {
-      const updatedImports = mergeImports(command.targetFile, parseSuccess.imports, command.imports)
+      // TODO handle duplicate name mapping
```

Comment:
are these `// TODO handle duplicate name mapping` still outstanding? do you want to fix these in a follow-up PR?

Reply:
yes, this PR introduces the mechansim change (inside `mergeImports` and other places), and also the usage of it in the `reparenting` flow, I didn't want to make it way too large, so I will have follow-up PRs for specific locations